Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1244,7 +1244,23 @@ void TextEditor::insert_at_cursor_or_replace_selection(const StringView& text)
     VERIFY(is_editable());
     if (has_selection())
         delete_selection();
+
+    // check if adding a newline leaves the previous line as just whitespace
```

Comment:
Comments should look like sentences, with proper punctuation (https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#comments)

```suggestion
    // Check if adding a newline leaves the previous line as just whitespace.
```